### DISCUSSION TITLE
Wiki Example For Azure Pipelines LSP

### DISCUSSION THREAD

**Discussion by RoastBeefer00** [helix-editor/helix#8863] [General]
Neovim's `Mason` has support for [Azure Pipelines LSP](https://www.npmjs.com/package/azure-pipelines-language-server) but I cannot find any documentation on how to set this up in Helix.  I write a lot of ADO pipelines for work and I'd love to have this LSP set up.

In additional having documentation of how to automate Helix knowing when to use YAML LSP vs ADO Pipeline LSP would be amazing.

**Comment by pascalkuthe** [comment]
The wiki is free to edit for anyone, if you have any suggestions feel free to add them.

**Comment by RoastBeefer00** [comment]
If I figure it out I'll gladly update the wiki.  I'm saying I don't know how to set up this LSP for Helix and I'd like to know how.
